.travis.yml

```diff
@@ -1,21 +1,37 @@
 sudo: required
 dist: trusty
 
+language: ruby
+
+cache:
+  bundler: true
+
+rvm:
+  - 2.3.3
+
+install:
+  - bundle install
+ 
+ jobs:
+  include:
+    script:
+       - bundle exec img_checker
```

Comment:
I mentioned to put `bundle exec img_checker` after `bundle install` so that you don't run it as separate job as you did here. The tests are not starting cause of intendation failure.

Please understand the working and commit the changes.
install section should look like
```
- bundle install
- bundle exec img_checker
```